Scripts/run_TestOneCase.sh

```diff
@@ -53,7 +53,7 @@ runEncoderCommandInital()
 					"-slcmd 0" "-slcnum 0" "-slcmd 1" "-slcnum 1"\
 					"-slcmd 2" "-slcnum 2" "-slcmd 3" "-slcnum 3"\
 					-nalsize \
-					-iper   -thread    -ltr \
+					-iper   -thread  " -loadbalancing 0 "  -ltr \
```

Comment:
need to add this argument during case generation.
   -loadbalancing  should be placed in  aEncoderCommandSet 
  and 
     0                     should be placed in aEncoderCommandValue

well, I will add it .
great, thanks!